docs/integrations/app-development/opentelemetry/puppet-opentelemetry.md

```diff
@@ -0,0 +1,234 @@
+---
+id: puppet-opentelemetry
+title: Puppet - OpenTelemetry Collector
+sidebar_label: Puppet - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Puppet.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/app-development/puppet.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
```

Comment:
This image link is broken when I run it in a local build. Verify that it works in your system.